internal/exec/describe_stacks.go

```diff
@@ -73,11 +73,54 @@ func ExecuteDescribeStacksCmd(cmd *cobra.Command, args []string) error {
 		sections = strings.Split(sectionsCsv, ",")
 	}
 
-	stacksMap, err := FindStacksMap(cliConfig)
+	finalStacksMap, err := ExecuteDescribeStacks(cliConfig, filterByStack, components, componentTypes, sections)
 	if err != nil {
 		return err
 	}
 
+	if format == "yaml" {
```

Comment:
This same code is in `ExecuteDescribeAffectedCmd`. Can we create a new function in a utils.go or equivalent and use the function in both places?